deploy/examples/object.yaml

```diff
@@ -108,6 +108,13 @@ spec:
   #zone:
   #name: zone-a
   # service endpoint healthcheck
+    # opsLogSidecar:
```

Comment:
This new section should be before lines 108-110. The settings on those lines do not belong to the `gateway`, but this `opsLogSidecar` does belong to `gateway`.